src/shared/components/dateRangePicker/DateRangePicker.tsx

```diff
@@ -70,6 +80,11 @@ class DateRangePicker extends PureComponent<Props, State> {
             onClick={this.handleSetTimeRange}
             text="Apply Time Range"
             testID="daterange--apply-btn"
+            status={
+              this.state.validDateRange
```

Comment:
we can destructure the `validDateRange` here:

https://github.com/influxdata/ui/pull/1079/files#diff-01e353f1c997af4eed6a066de1df2aa36349b7b10b77ea445251adfd797d0b0aR51

Reply:
✅